recipes-appfw/iot-app-fw/iot-app-fw_git.bb

```diff
@@ -10,6 +10,7 @@ SRC_URI = " \
     git://git@github.com/ostroproject/iot-app-fw.git;protocol=https;branch=kli/devel/1.x-fixes \
     file://80-container-host0.network \
     file://80-container-ve.network \
+    file://appfw-packet-forward.conf \
```

Comment:
The appfw-packet-forward.conf file seems to be missing?

Can you remove this commit and re-submit later so that we can focus on merging the release-critical bug fixes first?